### DISCUSSION TITLE
New badge ideas

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20526] [Other Feature Feedback, Questions, & Ideas]
* Number of builds on GitHub Actions
* Number of releases
* Number of (feature) branches
* Mentioned count
* Owner of a source repository with number of issues
* Lines of code committed
* Big commit
* Documentation writer
Would need satisfying names for each of these achievements.

**Comment by devkennyy** [comment]
I'd love to hear your thoughts on these names 

> Number of builds on GitHub Actions

Master Builder

> Mentioned count

Influencer

> Owner of a source repository with number of issues

Trouble Maker

> Big commit

Chunky Commit

**Reply by ghost** [reply]
These are funny names. Chunky commit caused me to laugh out loud. Maybe chunky boi

**Reply by devkennyy** [reply]
Chunky boi is hilarious! Where's Github Staff 😂
